### ISSUE TITLE
Test results on macOS 10.13.6

### ISSUE THREAD

**Issue by tbussmann** [alberti42/Volume-Control#4]
Here are some initial test results of 1.7.0 (17) on 10.13.6 (both on a clean installed VM and a system that was successfully using iTunes Volume Control 1.6.7 (255) before):
* The App does launch
* It shows an empty / blank menubar icon
* It is not able to read "(0%)" nor control the volume
* "Accessibility" permissions are manually granted and those of "iTunes Volume Control" removed in control panel (it does not ask for the permission interactively)
* The unsigned app requires right click / "open" to run for the first time if you have security set to "App Store and identified developers"
* The app does contain an provisioning profile that is about to expire 2021-06-26

**Comment by alberti42** [comment]
yes, it is very disappointing. What comes to mind is that in the older versions, Music was called iTunes. If this would be the only problem, it would not be hard to deal with, because the only change they made in the way to control the app was in changing its name.

> Am 25.06.2021 um 10:46 schrieb Tobias Bussmann ***@***.***>:
> 
> 
> Here are some initial test results of 1.7.0 (17) on 10.13.6 (both on a clean installed VM and a system that was successfully using iTunes Volume Control 1.6.7 (255) before):
> 
> The App does launch
> It shows an empty / blank menubar icon
> It is not able to read "(0%)" nor control the volume
> "Accessibility" permissions are manually granted and those of "iTunes Volume Control" removed in control panel (it does not ask for the permission interactively)
> The unsigned app requires right click / "open" to run for the first time if you have security set to "App Store and identified developers"
> The app does contain an provisioning profile that is about to expire 2021-06-26
I don't understand. I thought the app is not signed at all.

> codesign --verify --verbose /Applications/Volume\ Control.app/
/Applications/Volume Control.app/: code object is not signed at all
In architecture: x86_64

**Comment by Careless2000** [comment]
I concur with the findings of Tobias. And with Voice Control running the overall "standard" volume control is not working.
The old version "iTunes Volume control" requires 10.14
